storage/innobase/dict/dict0load.cc

```diff
@@ -164,6 +165,21 @@ dict_load_virtual_low(
 	ulint*		base_pos,
 	const rec_t*	rec);
 
+/** Load a table added column definition from a SYS_COLUMNS_ADDED record to dict_table_t.
+@return	error message
+@retval	NULL on success */
+static
+const char*
+dict_load_column_added_low(
+	dict_table_t*	table,		/*!< in/out: table, could be NULL */
+	mem_heap_t*		heap,		/*!< in/out: memory heap for temporary storage */
+	const rec_t*	rec,		/*!< in: SYS_COLUMNS_ADDED record */
+	dict_index_t*	sys_index,
+	table_id_t*		table_id_out,
+	ulint*			pos_out,
+	char**			def_val_out,
+	ulint*			def_val_len_out);
```

Comment:
Document each parameter with `@param` comments. Can you move the function definition here, so that we are not duplicating these lines? Currently we first have a declaration and then later the actual definition.